e2etest/zt_newe2e_s2s_test.go

```diff
@@ -1,13 +1,14 @@
 package e2etest
 
 import (
+	"strconv"
+
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/blob"
 	"github.com/Azure/azure-storage-azcopy/v10/common"
-	"strconv"
 )
 
 func init() {
-	suiteManager.RegisterSuite(&S2STestSuite{})
+	//suiteManager.RegisterSuite(&S2STestSuite{})
```

Comment:
Why are these commented in all the files

Reply:
commented as of now. To validate the new test case added.
Will remove these changes and create a new ticket for adding a TC for this scenario.